runtime/indent/asm.vim

```diff
@@ -0,0 +1,40 @@
+" Vim indent file
+" Language:             asm
+" Previous Maintainer:  Philip Jones <philj56@gmail.com>
+" Upstream: https://github.com/philj56/vim-asm-indent
+" Latest Revision:      2017-07-01
+
+if exists("b:did_indent")
+  finish
+endif
+let b:did_indent = 1
+
+setlocal indentexpr=GetAsmIndent()
+setlocal indentkeys=<:>,!^F,o,O
+
+if exists("*GetAsmIndent")
+  finish
+endif
+
+let s:cpo_save = &cpo
+set cpo&vim
+
+function s:buffer_shiftwidth()
+  return shiftwidth()
```

Comment:
you don't need a wrapper function around `shiftwidth()` here

Reply:
Done.